Casks/font/font-g/font-gd-highwaygothicja.rb

```diff
@@ -0,0 +1,18 @@
+cask "font-gd-highwaygothicja" do
+  version "112b2"
+  sha256 "7f9af8273b302fdd1dbd85cee8060833b8fd872d244686899b0c5c62c3ffc951"
+
+  url "https://542.jp/font/data/GDHighwayGoJA-Rev#{version}-OTF.zip"
+  name "GD-HighwayGothicJA"
+  name "GD-高速道路ゴシックJA"
+  homepage "https://542.jp/font/"
+
+  livecheck do
+    url :homepage
+    regex(/GDHighwayGoJA-Rev(\d+(?:b\d+))-OTF\.zip/)
```

Comment:
```suggestion
    regex(/GDHighwayGoJA[._-]Rev(\d+(?:\.\d+)*(?:b\d+)?)[._-]OTF\.zip/)
```

We typically use `[._-]` as separators to make the regex more robust against naming changes. Similarly, `(\d+(?:\.\d+)*)` can also match version numbers with dots, and we should make the build or beta tag optional.